creds/netrc.go

```diff
@@ -24,7 +28,17 @@ func ParseNetrc(osEnv config.Environment) (NetrcFinder, string, error) {
 
 	nrcfilename := filepath.Join(home, netrcBasename)
 	if _, err := os.Stat(nrcfilename); err != nil {
-		return &noFinder{}, nrcfilename, nil
+		// If on Windows, try _netrc instead
+		if runtime.GOOS == "windows" {
+			altFilename := filepath.Join(home, netrcAltBasename)
+			if _, errAlt := os.Stat(altFilename); errAlt == nil {
+				nrcfilename = altFilename
+			} else {
+				return &noFinder{}, altFilename, nil
+			}
```

Comment:
As a very minor stylistic suggestion, because we're returning from one branch, we can make this a bit shorter and also follow the general Go convention of handling errors first, e.g.:
```suggestion
			if _, errAlt := os.Stat(altFilename); errAlt != nil {
				return &noFinder{}, altFilename, nil
			}
			nrcfilename = altFilename
```

(I'm just thinking of the old "Effective Go" page in the [section](https://go.dev/doc/effective_go#if) where they say, "The code reads well if the successful flow of control runs down the page, eliminating error cases as they arise. Since error cases tend to end in `return` statements, the resulting code needs no `else` statements."  But this is definitely not a hard-and-fast rule, and I'm sure there are lots of counterexamples littering our code.)